pytest_django/db_reuse.py

```diff
@@ -111,6 +114,15 @@ def create_test_db_with_reuse(self, verbosity=1, autoclobber=False,
     if hasattr(self.connection.features, 'confirm'):
         self.connection.features.confirm()
 
+    if DJANGO_VERSION > (1, 7):
+        call_command(
+            'migrate',
+            verbosity=max(verbosity - 1, 0),
+            interactive=False,
+            database=self.connection.alias,
+            run_syncdb=True,
+        )
+
```

Comment:
The command in Django 1.8 looks a bit different: https://github.com/django/django/blob/stable/1.8.x/django/db/backends/base/creation.py#L360-L369:

        call_command(
            'migrate',
            verbosity=max(verbosity - 1, 0),
            interactive=False,
            database=self.connection.alias,
            test_flush=not keepdb,
        )

What do you think about using #261 (Django's `--keepdb`), which would cover Django 1.8+, and then this for Django 1.7.x?